relations/src/shielder/withdraw.rs

```diff
@@ -31,40 +43,115 @@ use crate::{environment::CircuitField, relation::GetPublicInput};
 ///    Merkle tree with `merkle_root` hash in the root
 /// It also includes two artificial inputs `fee` and `recipient` just to strengthen the application
 /// security by treating them as public inputs (and thus integral part of the SNARK).
+/// Additionally, the relation has one constant input, `max_path_len` which specifies upper bound
+/// for the length of the merkle path (which is ~twice the height of the tree).
```

Comment:
Either I'm misunderstanding something or the path is at most the height (or `log(leaves_count)`).

Reply:
corrected